compiler/one-cmds/onecc

```diff
@@ -207,6 +216,14 @@ def main():
     parser = _get_parser()
     args = _parse_arg(parser)
 
+    # TODO: keep running after 'init' command if possible
```

Comment:
I'd like to get your opinion about this. I think that to invoke `init` feature is here.

Reply:
Good position? or else?

Reply:
I thought `onecc` called init like,
```bash
$ onecc init name_to_create.cfg
```
Has it been changed?

Reply:
Do you think that `onecc init` looks more natural? If so, I'll fix.

Reply:
Current version (influeced by `onecc -C`)
```
$ onecc -I mycfg.cfg
# or
$ onecc --init mycfg.cfg
```

Reply:
`one init` +1 by @jyoungyun